src/editor.jsx

```diff
@@ -311,8 +315,22 @@ var imageUrlsFromContent = function(content) {
 
 var Editor = React.createClass({
     propTypes: {
-        imageUploader: React.PropTypes.func,
         apiOptions: ApiOptions.propTypes,
+        className: React.PropTypes.string,
+        content: React.PropTypes.string,
+        disabled: React.PropTypes.bool,
+        // We don't use EnabledFeatures.propTypes here because it requires the
```

Comment:
I was hoping that there was a way of saying like `.isNotRequired` that would override something that was `.isRequired` but couldn't find any such thing.

I suppose I could just take the `.isRequired` off of EnabledFeatures.propTypes, and then add back the `.isRequired` inline everywhere that it's currently used. Would that be good?